server/mcp_server.py

```
@@ -140,28 +153,79 @@ def _get_token() -> str:
-        except httpx.HTTPStatusError as exc:
-            code = exc.response.status_code
-            try:
-                detail = exc.response.json()
-            except Exception:
-                detail = {"error": exc.response.text[:500]}
-            raise ValueError(f"Aurora API returned {code}: {detail}")
-        return resp.json()
+            detail = exc.response.json()
+        except (ValueError, httpx.ResponseNotRead):
+            detail = {"error": exc.response.text[:500]}
+        raise ValueError(f"Aurora API returned {code}: {detail}")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)